setup.py

```diff
@@ -69,6 +82,7 @@ def run(self):
     packages=find_packages(exclude="tests"),
     description="Metric logger for ML projects.",
     long_description=open("README.rst", "r", encoding="UTF-8").read(),
+    install_requires=render,
```

Comment:
So I cannot say that `dvclive` has no dependencies anymore? :(

Reply:
As long as we can say it's "lightweight" and people are able to quickly install it on any system, I don't think we should worry about it.

Reply:
Yeah, I'm just joking. Though, to be honest, there was something magical to the "It has NO dependencies". :)